reaction.go

```diff
@@ -242,7 +242,11 @@ func (r reactionQueryBuilder) Get(params URLQueryStringer, flags ...Flag) (ret [
 
 	emojiCode := ""
 	if e, ok := r.emoji.(*Emoji); ok {
-		emojiCode = e.Name + ":" + e.ID.String()
+		if e.ID != 0 {
```

Comment:
Could you use e.ID.IsZero() instead?
just for consistency and feature behaviour patches